packages/react-dev-utils/ModuleScopePlugin.js

```diff
@@ -18,10 +18,11 @@ class ModuleScopePlugin {
 
   apply(resolver) {
     const { appSrc } = this;
-    resolver.plugin('file', (request, callback) => {
```

Comment:
should this old plugin api still be supported for users of old webpack verions? I think its a bad idea (webpack 3 users can use current version)